src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -55,13 +55,23 @@
 public final class FindBreakpointEvidenceSpark extends GATKSparkTool {
     private static final long serialVersionUID = 1L;
 
+    private static final String OUTPUT_ORDER_SHORT_NAME = "sort";
+    private static final String OUTPUT_ORDER_FULL_NAME = "outputSortOrder";
+
     @ArgumentCollection
     private final FindBreakpointEvidenceSparkArgumentCollection params =
             new FindBreakpointEvidenceSparkArgumentCollection();
 
     @Argument(doc = "sam file for aligned contigs", shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
             fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
-    private String outputSAM;
+    private String outputAssemblies;
```

Comment:
That would work. I guess when Ted created that variable it was meant to specify the SAM format. And to me SAM and BAM are the same format with a single exception of compression.